SU-HSDO/stanford_mrc
# Mrcd8 154 news display

# READY FOR REVIEW

# Summary
- News node display
- News list display as a block
- Missing blocks for newly enabled theme
- Configured rabbit hole module for news
- Mainly this is just applying the patterns and verifying it has a similar layout as the designs. Additional styling will be needed for view pagers and other unrelated components.

# Needed By (Date)
- 12/7

# Urgency
- Medium

# Steps to Test
0. pull latest form `mrc_blt` and do a composer update --prefer-source
1. `blt sync:db`
2. `drush updb -y`
3. create a news node
4. verify it looks kinda like https://projects.invisionapp.com/share/DCEGPUZS4#/screens/263924862
5. Go to the blocks page. add the block "MRC News: List" to a desired page.
6. go to that page and verify it looks kinda like https://projects.invisionapp.com/share/DCEGPUZS4#/screens/263931728
7. configure the news node to have a source link. view that node in an icognito window to see if it redirects to your configured url.
8. remove that source link and view the node in an icognito window again to see the node you created.


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
